lib/modules/dosomething/dosomething_campaign/dosomething_campaign.helpers.inc

```diff
@@ -99,7 +99,7 @@ function dosomething_campaign_load($node, $public = FALSE) {
   $campaign->title = $node->title;
```

Comment:
the wrapper was still being used in a few places and I didn't wanna make more work than what was necessary for now